astropy/coordinates/tests/test_frames.py

```diff
@@ -180,7 +184,9 @@ def test_frame_repr():
         assert repr(i3) == ('<ICRS Coordinate: (ra, dec, distance) in (deg, deg, kpc)\n'
                             '    [(1.1, -15.6, 11.0), (2.1, 17.1, 21.0)]>')
 
-
+@pytest.mark.xfail(WINDOWS and six.PY3,
+                   reason='Python 3.x on Windows rounds different from '
+                          'everything else for unknown reasons')
```

Comment:
Oh, [here it is](https://github.com/astropy/astropy/pull/3450)  I couldn't find it before because @eteq opened the original issue.